package.json

```diff
@@ -1,9 +1,9 @@
 {
-  "name": "node-nuodb",
-  "author": "NuoDB, Inc.",
+  "name": "nuodb",
+  "author": "Dassault Systèmes SE",
   "description": "The official NuoDB driver for Node.js. Provides a high-level SQL API on top of the NuoDB Node.js Addon.",
   "license": "Apache-2.0",
-  "version": "3.6.0",
+  "version": "3.7.0",
   "main": "index.js",
   "keywords": [
```

Comment:
Let's add more keywords for the driver. Examples:
database, SQL, driver

Reply:
Applied suggestion, driver was already there, but added the others.